packages/vitest/src/node/reporters/renderers/windowedRenderer.ts

```diff
@@ -54,6 +55,9 @@ export class WindowRenderer {
     restoreCursor()
     this.write(HIDE_CURSOR, 'output')
 
+    // Write buffered content on unexpected exits, e.g. direct `process.exit()` calls
+    onExit(() => this.flushBuffer())
```

Comment:
doesn't `process.on('exit')` work? 🤔

Reply:
`exit` is not called on SIGTERM, e.g. `ctrl + c`. 

But now we can also replace `restore-cursor` with direct usage of `signal-exit`.

Reply:
`process.on('SIGTERM')` should be called on SIGTERM 🤔

Reply:
I just don't like overriding `process.reallyExit`

Reply:
`onExit` watches these signals: 

```
signals.push('SIGHUP', 'SIGINT', 'SIGTERM');
29if (process.platform !== 'win32') {
30    signals.push('SIGALRM', 'SIGABRT', 'SIGVTALRM', 'SIGXCPU', 'SIGXFSZ', 'SIGUSR2', 'SIGTRAP', 'SIGSYS', 'SIGQUIT', 'SIGIOT'
31    // should detect profiler and enable/disable accordingly.
32    // see #21
33    // 'SIGPROF'
34    );
35}
36if (process.platform === 'linux') {
37    signals.push('SIGIO', 'SIGPOLL', 'SIGPWR', 'SIGSTKFLT');
38}
```